### PR TITLE
Work Edits

### PR DESCRIPTION
**Pull request by deutschbitte**

### CODE REVIEW

File: work/index.html [hunk 1]
```diff
@@ -42,28 +42,28 @@ <h1 class="logo"><a href="../welcome/">smeidu</a></h1>
...
-		<h3 class="upperh3">Recent clients (2019 - 2022)</h3>
+		<h3 class="upperh3">Recent Clients (2019–2022)</h3>
 		<ul>
-		<li>Google - <a href="https://safety.google/intl/en-en/security/security-tips/">Security UX</a> (6 month, 100%)</li>
-		<li>Twaice - <a href="https://twaice.com/platform/">Battery analytics platform</a> (5 month, 50%)</li>
-		<li>EnergieAG - <a href="https://www.wertstatt8.at/">Innovation Lab</a> (7 month, 50%)</li>
-		<li>Deutsche Telekom - <a href="https://www.smarthome.de/">Smart Home UX</a> (6 month, 100%)</li>
-		<li>Clara - <a href="https://alles-clara.at/">Co-PO & UX Lead</a> (16 month, 75%)</li>
-		<li>Creative Region - <a href="https://creativeregion.org/2021/03/mentoring-fuer-creatives/">Mentoring</a> (6 month)</li>
+		<li>Google — <a href="https://safety.google/intl/en-en/security/security-tips/">Security UX</a> (6 months, 100%)</li>
```
Comment (deutschbitte) [inline work/index.html:61]: I'm curious about your decision to list the time frame and the percentage of work. I personally wouldn't include it, but it's up to you (if this is something you've seen elsewhere, or information you think prospective clients find interesting, etc)

Comment (smeidu) [reply work/index.html:61]: started as way of keeping track what I ve worked on as freelancer, mostly for myself I guess.
you are right about keeping such things out. makes it easier and cleaner!

File: work/index.html [hunk 2]
```diff
@@ -42,28 +42,28 @@ <h1 class="logo"><a href="../welcome/">smeidu</a></h1>
 <section id="Contact">
     <h2 class="nodisplay">Work</h2>
 	<div class="six colums"><img src="studio.png"></div>
-	<div class="six colums"><p class="movieintro addEMmargtop">I started my own studio in 2019, working for selected clients and also starting my own efforts here and there.</p><p class="movieintro">Interested in working with me? Or just having a coffee chat? <a href="../contact/">Contact me</a></p></div>
+	<div class="six colums"><p class="movieintro addEMmargtop">I started my own studio in 2019, working for selected clients and also starting my own efforts here and there.</p><p class="movieintro">Interested in working with me, or just having a coffee chat? <a href="../contact/">Contact me</a>.</p></div>
```
[the remark this answers is not in the document]
Comment (smeidu) [reply work/index.html:45]: a mixture of both. also a way to communicate that I am interested in (co)founding another startup in the future.

[the remark this answers is not in the document]
Comment (deutschbitte) [reply work/index.html:45]: Hmmm, maybe something like

"I started my own studio in 2019, working for selected clients and occasionally pursuing topics of special interest."

"I started my own studio in 2019, working for selected clients and occasionally starting projects relating to my interests."

Or similar. Thoughts? Suggestions?

[the remark this answers is not in the document]
Comment (smeidu) [reply work/index.html:45]: sounds too much like "hobby projects" to me. it should be about professional, but non client projects: open source projects, art projects, exploring ideas that eventually could lead to another startup.

[the remark this answers is not in the document]
Comment (smeidu) [reply work/index.html:45]: maybe something in that direction?
"I started my own studio in 2019, working for selected clients and also starting self-initiated projects here and there"

[the remark this answers is not in the document]
Comment (smeidu) [reply work/index.html:45]: cool. SGTM

**Comment by deutschbitte** [comment]
If you're happy with this we can merge and see how it looks

**Pull request merged**